node/node_newblock.go

```diff
@@ -145,19 +158,18 @@ func (node *Node) proposeNewBlock() (*types.Block, error) {
 	receiptsList := node.proposeReceiptsProof()
 	if len(receiptsList) != 0 {
 		if err := node.Worker.CommitReceipts(receiptsList); err != nil {
-			utils.Logger().Error().Err(err).Msg("[proposeNewBlock] cannot commit receipts")
+			return nil, err
```

Comment:
what is the reason for this new behavior? failed to commit receipts stop from proposing new block?

Reply:
yep, if anything fail to commit, there is no point of continuing.